docs/Q-and-A.md

```diff
@@ -0,0 +1,649 @@
+#### Version Segment Definitions:
+1. Major: Major protocol evolution, with breaking protocol advancements so large they warrant incrementing the major version.
+2. Minor: Critical updates, protocol forking changes, or security patches that require all nodes to upgrade.
+3. Patch: Non-critical changes that do not require nodes to upgrade.
+
+#### New Version Activation
+New versions of the protocol, or modifications to parameter values by implementers, muat be activated at a specified _Ledger Time_ so **all nodes can remain in sync by enforcing the same ruleset and parameters** beginning at the same logical starting point.\ 
+All transactions that occur after the specified Ledger Time will adhere to the associated version’s rules and parameters until a newer version of the protocol is defined and implemented at a future Ledger Time.\
+New Version Activation is **necessary for Minor and Major version segments**; the ones with breaking protocol advancements, and protocol forking changes, or security patches.
+_(@henkvancann)_
+
+## Could Sidetree work with be pruned or charded blockchains?
+No, any node needs a full copy of the blockchain that serves as a settlement layer 1. The reason is ordering of operation on DID is done via consensus of the blockchain.\
+_(@henkvancann)_
+## How to bootstrap Sidetree on the internet? Is it like fax machine; the more there are the more effective it is?
+It's bootstrapped by the first DID method that uses Sidetree under the hood. The more use of those DIDs the more Sidetree gets used.  However, there is no network effect as such from extended use. None-the-less there is a huge network effect when mulitple DID methods are running on Sidetree, because they share the same resolution, protocol and file structure in the CAS.
+_(@henkvancann)_
+
+## What do you need to purchase to operate Sidetree or on sidetree based DIDs like ION?
+
+But to look up the IDs in ION, you have to have a full node, you've got to have the index of history. So in fact, we're going to be spawning a new reason why people will want to run full nodes, because it actually provides you concrete value you absolutely cannot get any other way. So running a node for us, it's a priority that anyone be able to do it.
+
+So our target hardware has been this little 2017 Intel NUC that I bought for $400, back three years when we started this. This thing is pretty underpowered, it's like a 2017 i5, it's got six gigs of RAM, a half a terabyte hard drive and we're running this thing at full scale, as fast as you could do, even our Azure blades or anything like that. You could run it at home right under your desk! Part of that is running a Bitcoin full node, IPFS, and other things, so anyone should be able to get it started.\
+[Source: What bitocin Did 231](https://www.whatbitcoindid.com/wbd231-daniel-buchner)
+
+## What do you need to do?
```

Comment:
Im confused by who the "you" refers to in this case

Reply:
rephrased this, thx